EventFilter/L1TRawToDigi/plugins/implementations_stage2/RegionalMuonGMTUnpacker.h

```diff
@@ -10,10 +10,13 @@ namespace l1t {
       class RegionalMuonGMTUnpacker : public Unpacker {
          public:
             bool unpack(const Block& block, UnpackerCollections *coll) override;
+            void setKalmanAlgoTrue() { isKalman_ = true; }
```

Comment:
Is there never going to be a need for a setKalmanAlgoFalse?

Reply:
I initialise the object to isKalman_ = false in [1], similar to the way this is done for the BMTF[2] for consistency. I think once one has made the choice that it should be unpacking Kalman muons one won't need to switch it back off, right?

[1] https://github.com/dinyar/cmssw/blob/788557afd732cfc0c4afa1c30d10f6e4c8aff99a/EventFilter/L1TRawToDigi/plugins/implementations_stage2/RegionalMuonGMTUnpacker.h#L19
[2] https://github.com/dinyar/cmssw/blob/788557afd732cfc0c4afa1c30d10f6e4c8aff99a/EventFilter/L1TRawToDigi/plugins/implementations_stage2/BMTFUnpackerOutput.h#L14

Reply:
That was my guess as well. As long as all processors use the same FW that should be OK then.